documentation/source/user_section/tutorials/registration-to-template/lumbar-registration/template-registration.rst

```diff
@@ -0,0 +1,36 @@
+.. _applying-lumbar-registration-algorithm:
+
+Applying the registration algorithm
+###################################
+
+To apply the registration algorithm, the following command is used:
+
+.. code:: sh
+
+   sct_register_to_template -i t2_crop.nii.gz \
+                            -s t2_crop_seg.nii.gz \
+                            -ldisc t2_crop_labels.nii.gz \
+                            -c t2 -qc ~/qc_singleSubj \
+                            -param step=1,type=seg,algo=centermassrot:step=2,type=seg,algo=bsplinesyn,metric=MeanSquares,iter=3,slicewise=0:step=3,type=im,algo=syn,metric=CC,iter=3,slicewise=0
+
```

Comment:
The instructions for updating the PAM50 template are four pages away, and easy to overlook. I would suggest adding the exact error message here, close to where it's likely to happen, with a link to the instructions.

```suggestion

.. note::

   If you get the following error message, it means that your installed version of the PAM50 template is too old:
   
   .. code::
   
      ERROR: Wrong landmarks input. Labels must have correspondence in template space. 
      Label max provided: 60
      Label max from template: 21   

   In that case, you should update it by following `these instructions <before-starting-lumbar-registration>`_.
```

Reply:
Lovely suggestion! I very much think this is a good idea.